Packs/GoogleCloudSCC/Integrations/GoogleCloudSCC/GoogleCloudSCC.py

```diff
@@ -0,0 +1,1125 @@
+
+
+def prepare_human_readable_dict_for_list_asset(asset: Dict[str, Any]) -> Dict[str, Any]:
+    """
+    Prepare human-readable dictionary for list asset command.
+    :param asset: asset information
+    :return: human-readable dict
+    """
+    asset_url = GoogleNameParser.get_asset_url(asset.get("name", ""))
+
+    return {
+        COMMON_STRING["RESOURCE_NAME"]: asset.get("securityCenterProperties", {}).get("resourceName", ""),
+        "Resource Type": asset.get("securityCenterProperties", {}).get("resourceType", ""),
+        "Resource Owners": asset.get("securityCenterProperties", {}).get("resourceOwners", {}),
+        "Project": asset.get("resourceProperties", {}).get("name", ""),
+        "Name": MARKDOWN_LINK.format(asset.get("name", ""), asset_url),
+        COMMON_STRING["SECURITY_MARKS"]: asset.get("securityMarks", {}).get("marks", {})
+    }
+
+
+def prepare_outputs_for_list_assets(result) -> Tuple[Dict[str, Any], str]:
+    """
+    Preparing context output and human-readable for list-assets command.
+
+    :param result: result dictionary
+    :return:
+    """
+    hr_asset_list = []
+    ec_asset_list = []
+
+    list_assets = result.get("listAssetsResults", [])
+    if len(list_assets) <= 0:
+        return {}, ERROR_MESSAGES["NO_RECORDS_FOUND"].format("asset")
+
+    read_time = result.get("readTime", "")
+    total_size = result.get("totalSize")
+
+    for asset in list_assets:
+        flatten_keys_to_root(asset, ["asset"], {"readTime": read_time, "stateChange": asset.get("stateChange", None)})
+        ec_asset_list.append(asset)
+        hr_asset_dict = prepare_human_readable_dict_for_list_asset(asset)
+        hr_asset_list.append(hr_asset_dict)
+
+    # Preparing headers
+    headers = ["Name", "Project", COMMON_STRING["RESOURCE_NAME"], "Resource Type", "Resource Owners",
+               COMMON_STRING["SECURITY_MARKS"]]
+    readable_output = tableToMarkdown(GET_OUTPUT_MESSAGE["HEADER_MESSAGE"].format("asset(s)", total_size),
+                                      t=hr_asset_list, headers=headers, removeNull=True)
+
+    # preparing context
+    ec_asset_dict: Dict[str, Any] = {
+        OUTPUT_PREFIX["LIST_ASSET"]: ec_asset_list
+    }
+
+    next_page_token = result.get("nextPageToken", "")
+    if next_page_token:
+        token_ec = {"name": "google-cloud-scc-asset-list", "nextPageToken": next_page_token}
+        ec_asset_dict.update({OUTPUT_PREFIX["TOKEN"]: token_ec})
+        readable_output += NEXT_PAGE_TOKEN.format(next_page_token)
+
+    return remove_empty_elements(ec_asset_dict), readable_output
+
+
+def flatten_keys_to_root(data_dict: Dict[str, Any], keys: List, update_dict: Dict[str, Any]):
+    """
+    Add list of keys to root level in dict
+
+    :param data_dict: dictionary
+    :param keys: list of keys
+    :param update_dict: dictionary that will be add in data dict
+    :return: flatten dict for provided keys
+    """
+    for key in keys:
+        value = data_dict.pop(key, keys)
```

Comment:
why is the default value is `keys`?

Reply:
Yes, now keys is replaced by None and we have also added unit test cases for the same.